server/filters/convert-image-uri.js

```diff
@@ -0,0 +1,75 @@
+
+export default function convertImageUri(originalUri, requiredSize, useIiif, useIiifOrigin) {
+  const imageMap = {
+    wordpress: {
+      root: 'https://wellcomecollection.files.wordpress.com/',
+      iiifRoot: 'https://iiif.wellcomecollection.org/image/wordpress:',
+      iiifOriginRoot: 'https://iiif-origin.wellcomecollection.org/image/wordpress:'
+    },
+    prismic: {
+      root: 'https://prismic-io.s3.amazonaws.com/wellcomecollection/',
+      imigixRoot: 'https://wellcomecollection-prismic.imgix.net',
```

Comment:
imgX is not going to be a backup system - we want to remove it completely.
The backup would be to serve straight from S3.

Reply:
Only intended this to be a very temporary fallback until sure loris is ok and then would remove when removing the useIiif flag